esileapclient/tests/functional/v1/test_basic.py

```diff
@@ -213,6 +215,38 @@ def test_lease_show_offer_deleted(self):
         details = esi.lease_show(self.clients['child-lessee'], lease['uuid'])
         self.assertEqual(details['status'], 'deleted')
 
+    @unittest.skipIf('OS_FUNCTIONAL_NODE_[...]' not in os.environ.keys(),
+                     "env variable to test node list function is not set")
+    def test_node_list_long(self):
+        """ Tests functionality of "esi node list" using node_uuid.
+            checks node_uuid is present in node list or not.
+            Test steps:
+            1) Set the environment variable using
+               export OS_FUNCTIONAL_NODE_IDENT_UUID = node_uuid
+            2) It check that the output of "node list" contains
+               the node uuid it's tested with. """
+
+        node_uuid = os.getenv('OS_FUNCTIONAL_NODE_IDENT_UUID')
+        listings = esi.node_list(self.clients['admin'], long=True)
+        self.assertNotEqual(listings, [])
+        self.assertIn(node_uuid, [x['UUID'] for x in listings])
+
+    @unittest.skipIf('OS_FUNCTIONAL_NODE_[...]' not in os.environ.keys(),
+                     "env variable to test node list function is not set")
```

Comment:
The change to the skip reason made to the previous function should also be made here.

Reply:
Yes.